catalog/ui/catalog-ui-search/src/main/webapp/component/golden-layout/golden-layout.hbs

```diff
@@ -17,7 +17,7 @@
 <div class="golden-layout-toolbar">
     <span></span>
     <div class="toolbar-actions">
-        <div class="to-add" title="Add visualization" data-help="Add visualiation">
+        <div class="to-add" title="Add visualization" data-help="Add a new visualization tab to the current view. Using a variety visualizations can help when looking at results.">
```

Comment:
✏️ `... Using a variety of visualizations...`.

Reply:
May be clearer also noting also that visualizations are used to view Metacard results in different ways.

Reply:
Good catch